tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t1.py

```diff
@@ -0,0 +1,668 @@
+                ),
+                validators.if_then_validator(
+                    condition_field="SANC_REDUCTION_AMT",
+                    condition_function=validators.isLargerThan(0),
+                    result_field="OTHER_SANCTION",
+                    result_function=validators.oneOf((1, 2)),
+                ),
+                validators.if_then_validator(
+                    condition_field="OTHER_TOTAL_REDUCTIONS",
+                    condition_function=validators.isLargerThan(0),
+                    result_field="FAMILY_CAP",
+                    result_function=validators.oneOf((1, 2)),
+                ),
+                validators.if_then_validator(
+                    condition_field="OTHER_TOTAL_REDUCTIONS",
+                    condition_function=validators.isLargerThan(0),
+                    result_field="REDUCTIONS_ON_RECEIPTS",
+                    result_function=validators.oneOf((1, 2)),
+                ),
+                validators.if_then_validator(
+                    condition_field="OTHER_TOTAL_REDUCTIONS",
+                    condition_function=validators.isLargerThan(0),
+                    result_field="OTHER_NON_SANCTION",
+                    result_function=validators.oneOf((1, 2)),
+                ),
+                validators.sumIsLarger(
+                    (
+                        "AMT_FOOD_STAMP_ASSISTANCE",
+                        "AMT_SUB_CC",
+                        "CASH_AMOUNT",
+                        "CC_AMOUNT",
+                        "TRANSP_AMOUNT",
+                        "TRANSITION_SERVICES_AMOUNT",
+                        "OTHER_AMOUNT",
+                    ),
+                    0,
+                ),
+            ],
+            fields=[
+                Field(
+                    item="0",
+                    name="RecordType",
+                    friendly_name="record type",
+                    type="string",
+                    startIndex=0,
+                    endIndex=2,
+                    required=True,
+                    validators=[],
+                ),
+                Field(
+                    item="4",
+                    name="RPT_MONTH_YEAR",
+                    friendly_name="reporting month year",
+                    type="number",
+                    startIndex=2,
+                    endIndex=8,
+                    required=True,
+                    validators=[
+                        validators.dateYearIsLargerThan(1998),
+                        validators.dateMonthIsValid(),
+                    ],
+                ),
+                Field(
+                    item="6",
+                    name="CASE_NUMBER",
+                    friendly_name="case number",
+                    type="string",
+                    startIndex=8,
+                    endIndex=19,
+                    required=True,
+                    validators=[validators.isAlphaNumeric()],
```

Comment:
also need a better validator for this field. [see comment here](https://github.com/raft-tech/TANF-app/issues/2681#issuecomment-1852328435)